downstream/modules/devtools/proc-rhdh-firewall-example-edit.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="rhdh-firewall-example-edit_{context}"]
+= Editing your firewall playbook project
+
+The Ansible plug-ins integrate OpenShift Dev Spaces to edit your Ansible projects.
+OpenShift Dev Spaces provides on-demand, web-based Integrated Development Environments (IDEs).
+
+Ansible Git projects provisioned using the Ansible plug-ins include best practice configurations for OpenShift Dev Spaces.
+These configurations include installing the Ansible VS Code extension and providing access from the IDE terminal to Ansible development tools,
+such as Ansible Navigator and Ansible Lint.
+
+[NOTE]
+====
+OpenShift DevSpaces is optional and it is not required to run the Ansible plug-ins.
```

Comment:
I noticed there is no space between Dev and Spaces here but there is in other mentions of it, should it be consistent?